src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -49,6 +49,16 @@ public without sharing class UTIL_CustomSettingsFacade {
     static List<Relationship_Auto_Create__c> relationshipAutocreate;
     static List<Relationship_Lookup__c> relationshipLookup;
 
+    // Constructor
+    public UTIL_CustomSettingsFacade() {
+        User usr = [SELECT ProfileId FROM User where Id = :UserInfo.getUserId()]; 
+        Profile prof = [SELECT PermissionsCustomizeApplication FROM Profile Where Id =: usr.ProfileId];
+        System.debug('PermissionsApiEnabled: '+prof.PermissionsApiEnabled);
+
+        throw new StgException('You do not have permission to Customize this Application.');
```

Comment:
I need to remove this. I accidentally committed in progress work from another branch...